runelite-client/src/main/java/net/runelite/client/plugins/boosts/BoostsConfig.java

```diff
@@ -61,12 +61,12 @@ default boolean useRelativeBoost()
 	}
 
 	@ConfigItem(
-		keyName = "displayIndicators",
-		name = "Display as indicators",
-		description = "Configures whether or not to display the boost as indicators",
+		keyName = "displayInfoboxes",
```

Comment:
Changing the key name will break existing configs. I'd recommend not changing this.